packages/cli-repl/src/mongosh-repl.ts

```diff
@@ -658,8 +730,39 @@ class MongoshNodeRepl implements EvaluationListener {
     input: string,
     filename: string
   ): Promise<ShellResult> {
-    const { repl } = this.runtimeState();
-    return await promisify(repl.eval.bind(repl))(input, repl.context, filename);
+    const { repl, context } = this.runtimeState();
+    if (repl) {
+      return await promisify(repl.eval.bind(repl))(input, context, filename);
+    } else {
```

Comment:
Suuuper nitpicky and I'm surprised that our linting didn't complain, but you can probably just remove the else and save one level of indentation because the if is guaranteed to return. But also just ignore me if you prefer this style.

Reply:
Yeah, thanks for pointing this out! Addressed this in 8f7f38f :)